tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -259,6 +259,11 @@ def get_modified_metadata_files(modified_files: Set[Union[str, Path]]) -> Set[Pa
     }
 
 
+def get_expected_metadata_files() -> Set[Path]:
+    changed_connectors = get_changed_connectors()
+    return {changed_connector.metadata_file_path for changed_connector in changed_connectors}
```

Comment:
> Or is it the case that modifying a metadata file automatically makes that connector a changed connector because the change happened in that connector's directory?

This is what happens.

Reply:
Perfect. Thanks!